docs/guides/honeypot.md

```diff
@@ -173,13 +174,19 @@ Now you are ready to start ContainerSSH:
 
 ```
 docker run -d \
-  --restart=always \
+  --restart=always --name containerssh \
```

Comment:
This could fix unrecognized domain name error with MinIO:
>{"timestamp":"2022-06-24T18:02:00Z","level":"error","code":"AUDIT_S3_SINGLE_UPLOAD_FAILED","message":"single upload failed for audit log 24b7097af4a44237bfc44652ca5b8614 (RequestError: send request failed\ncaused by: Put "http://honeypot-bucket.10.0.0.11:9000/24b7097af4a44237bfc44652ca5b8614\": dial tcp: lookup honeypot-bucket.10.0.0.11 on 127.0.0.53:53: no such host)","details":{"module":"audit"}}